src/frontend/ui/commun/IndicateurGraphique/IndicateurGraphique.module.css

```
@@ -4,8 +4,11 @@
 
 .titraille {
   margin-bottom: .5rem;
-}
-
-.titraille abbr {
   margin-right: .5rem;
 }
+
+.mise-à-jour-source {
+  align-items: end;
+  display: flex;
+  flex-wrap: wrap;
+}
```

```suggestion
.titraille {
  margin-bottom: 0;
  margin-right: .5rem;
}

.mise-à-jour-source {
  align-items: center;
  display: flex;
  flex-wrap: wrap;
}
```
Parce que ce n'était pas bien aligné verticalement sur mon Firefox.